tests/integration/components/sl-radio-group-test.js

```diff
@@ -0,0 +1,165 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-radio-group', 'Integration | Component | sl radio group', {
+    integration: true
+});
+
+test( 'The disabled state applies the disabled attribute and class', function( assert ) {
+    this.set( 'testName', 'test' );
```

Comment:
Right. Does not using name="testName" work, without needing to set the testName property? name is a string - doesn't have to be a bound value.